packages/block-library/src/separator/deprecated.js

```diff
@@ -49,6 +49,7 @@ const v1 = {
 			style: customColor
 				? { color: { background: customColor } }
 				: undefined,
+			tagName: 'hr',
```

Comment:
This isn't correct. The deprecation attributes should match definitions at the time of creation.

Reply:
I think you're misunderstanding the purpose of this function. This function is supposed to migrate the attributes to the correct/latest version. And in the latest version the tagName need to be defined in the block's attributes (because it has a default value)

Reply:
Riad is right about this. We need to add this.